TombEngine/Specific/level.cpp

```diff
@@ -1517,6 +1519,33 @@ void LoadBoxes()
 	}
 }
 
+void LoadMirrors()
+{
+	int mirrorCount = ReadCount();
+	TENLog("Mirror count: " + std::to_string(mirrorCount), LogLevel::Info);
+	g_Level.Mirrors.reserve(mirrorCount);
+
+	for (int i = 0; i < mirrorCount; i++)
+	{
+		auto& mirror = g_Level.Mirrors.emplace_back();
+
+		mirror.RoomNumber = ReadInt16(); // TODO: Write Int32 to level instead. Short isn't used for room numbers anymore.
```

Comment:
Is it necessary for this to be loaded as a `short`?

Reply:
No, but I guess it's being done out of habit. I think that all level file structs still use short for room number, because they used it in original.